CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 First alpha release.
 
+[See migration guide for details upgrading from 2.x to 3.x](https://github.com/auraphp/Aura.Sql/blob/695b240f1bd64dd5ba457935d7c0337be1055be5/docs/upgrade.md)
```

Comment:
That's fine; I think it goes well with that particular changelog entry. Having said that, a "final" conversion guide should be part of the 3.0.0 stable release.